2018-oscon/machine-learning-on-source-code.md

```diff
@@ -0,0 +1,44 @@
+# Machine Learning on Source Code: understanding large codebases
+
+**Updates**:  https://github.com/src-d/conferences/issues/9
+
+**Sent on**:  2018/01/30
+
+**Status**:   proposed
+
+**Author**:   Francesc Campoy
+
+**Slides**:   TBA
+
+**Proposal**: N/A
+
+**Description**:
+
+Machine Learning has the power of automating human activities. From spam detection to autonomous driving we benefit greatly.
+
+But what about reviewing code? Or maybe even coding!
+Not only we think this is possible but we're working on it and the whole project is open sourced.
+
+This talk explains how we're tackling this problem, the current state, and what we predict for years to come.
+
+**Abstract**:
+
+This talk is an introduction to the topic of Machine Learning on Source Code: a term coined at source{d} to describe the application of techniques from Natural Language Processing, Source Code Analysis, and Big Data to large code bases.
```

Comment:
One comment for the future, I am not sure if we should position our talks about the company but instead purely about MLoSC itself.

Reply:
I'm not sure what you mean here. Do you think I should just drop the reference to source{d} or should I make it more prominent?